packages/nodejs/.changesets/add-opentelemetry-span-processor.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "minor"
+type: "add"
+---
+
+Add OpenTelemetry span processor
```

Comment:
This can use a bit more explanation what this means for the user, if it means anything. If users cannot use it yet, let's remove the changeset for now.